### ISSUE TITLE
docs: no examples of call overloading

### ISSUE THREAD

**Issue by chrishna1** [JuliaLang/julia#26274]

**Comment by fredrikekre** [comment]
Could you be more specific? Is there a section that exist for the stable docs that is missing from the latest docs? Or are you simply requesting more documentation for this?

**Comment by chrishna1** [comment]
It is missing in both stable and latest docs but there was one example in 0.4.8 pre documentation.

**Comment by StefanKarpinski** [comment]
https://docs.julialang.org/en/stable/manual/methods/

**Comment by KristofferC** [comment]
Was this issue regarding call overloading perhaps?

**Comment by chrishna1** [comment]
Yes. I meant call overloading. I thought they can be used interchangeably.

**Comment by JeffBezanson** [comment]
There is an example under the heading "Function-like objects" in the methods chapter.

**Comment by fingolfin** [comment]
Given that there is indeed such an example in https://docs.julialang.org/en/v1/manual/methods/#Function-like-objects we can close
